packages/teams-js/CHANGELOG.md

```diff
@@ -273,45 +46,54 @@ Tue, 01 Mar 2022 19:50:49 GMT
+      - `settings.getSettings` has been renamed `pages.getConfig`
+    - Added `pages.navigateToApp` that navigates to the given application ID and page ID, with optional parameters for a WebURL (if the application cannot be navigated to, such as if it is not installed), Channel ID (for applications installed as a channel tab), and sub-page ID (for navigating to specific content within the page).
     - The following APIs have been been renamed and moved from `publicAPIs` to a new Pages.AppButton sub-capability in the new `pages.appButton` namespace:
       - `registerAppButtonClickHandler` has renamed and moved to `pages.appButton.onClick`
       - `registerAppButtonHoverEnterHandler` has renamed and moved to `pages.appButton.onHoverEnter`
       - `regsiterAppButtonHoverLeaveHandler` has renamed and moved to `pages.appButton.onHoverLeave`
     - The following APIs have been moved to a new Pages.BackStack sub-capability in the new `pages.backStack` namespace:
-      - `registerBackButtonHandler`
-      - `navigateBack`
+      - `registerBackButtonHandler` moved from `publicAPIs`
+      - `navigateBack` moved from `navigation` namespace
     - The following APIs have been renamed and moved into the Pages.Config sub-capability in the `pages.config` namespace (formerly the `settings` namespace):
       - `registerEnterSettingsHandler` has renamed and moved to `pages.config.registerChangeConfigHandler`
-      - `getSettings` has been renamed `pages.config.getConfig`
+      - `registerOnSaveHandler`
+      - `registerOnRemoveHandler`
       - `setSettings` has been renamed `pages.config.setConfig`
+      - `setValidityState`
     - The following APIs have been been moved from `privateAPIs` to a new Pages.FullTrust sub-capability in the new `pages.fullTrust` namespace:
       - `enterFullscreen`
       - `exitFullscreen`
     - The following APIs have been been moved to a new Pages.Tabs sub-capability in the new `pages.tabs` namespace:
-      - `getTabInstances`
-      - `getMruTabInstances`
-      - `navigateToTab`
-
-  - Added Dialog capability, renamed `tasks` namespace to `dialog`, and renamed APIs
-
-    - The following APIs have been renamed:
-      - `startTask` has been renamed `dialog.open`
-      - `submitTasks` has been renamed `dialog.submit`
-      - `updateTask` has been renamed `dialog.resize`
-      - `TaskInfo` interface has been renamed `DialogInfo`
-      - `TaskModuleDimension` enum has been renamed `DialogDimension`
+      - `getTabInstances` moved from `publicAPIs`
+      - `getMruTabInstances` moved from `publicAPIs`
+      - `navigateToTab` moved from `navigation` namespace
+
+  - Tasks APIs renamed and reorganized under new Dialog capability:
+
+    - Added `dialog` capability, which has support for HTML-based dialogs, and a `dialog.bot` sub-capability has been added for bot-based dialogs. At this time, `dialog` does not support adaptive card-based dialogs.
```

Comment:
@TrevorJoelHarris I'd love to have you take a look at this section in particular.

Reply:
this looks ok to me